test/harvest/test_merge_pubs.py

```diff
@@ -186,7 +193,7 @@ def test_merge(tmp_path, sul_pubs_csv, openalex_pubs_csv, dimensions_pubs_csv):
     assert output.is_file(), "output file has been created"
     df = pl.read_parquet(output)
     assert df.shape[0] == 5
-    assert df.shape[1] == 23
+    assert df.shape[1] == 25
```

Comment:
OpenAlex output includes two new columns `best_oa_location` and `citation_normalized_percentile`.